lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -769,6 +773,26 @@ def cmd_irb_help
   def cmd_irb(*args)
     defanged?
 
+    expressions = []
+
+    # Parse the command options
+    @@irb_opts.parse(args) do |opt, idx, val|
+      case opt
+        when '-e'
+          expressions << val
+        when '-h'
+          cmd_irb_help
+          return false
+      end
+    end
+
+    if !expressions.empty?
```

Comment:
unless is more idiomatic here since there's no else.  Also the return at the end isn't necessary.  I'll fix these on land